smoke-test/src/test/java/org/opennms/smoketest/minion/NMS14655_IT.java

```diff
@@ -0,0 +1,71 @@
+package org.opennms.smoketest.minion;
+
+import static com.jayway.awaitility.Awaitility.await;
+import static java.util.concurrent.TimeUnit.MINUTES;
+import static java.util.concurrent.TimeUnit.SECONDS;
+
+import java.io.PrintStream;
+import java.net.InetSocketAddress;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.opennms.smoketest.containers.MinionContainer;
+import org.opennms.smoketest.junit.MinionTests;
+import org.opennms.smoketest.stacks.MinionProfile;
+import org.opennms.smoketest.stacks.StackModel;
+import org.opennms.smoketest.utils.SshClient;
+import org.testcontainers.containers.wait.strategy.AbstractWaitStrategy;
+
+@Category(MinionTests.class)
+public class NMS14655_IT {
+    private static final String USERNAME = "admin";
+    private static final String PASSWORD = "admin";
+
+    @ClassRule
+    public static final MinionContainer MINION_CONTAINER = new MinionContainer(StackModel.newBuilder().build(), MinionProfile.newBuilder()
+            .withId("00000000-0000-0000-0000-000000ddba11")
+            .withLocation("Fulda")
+            .withDominionGrpcScvClientSecret("foobar")
+            .withWaitStrategy(c -> new AbstractWaitStrategy() {
+                @Override
+                protected void waitUntilReady() {
+                }
+            })
+            .build());
+
+    @Test
+    public void verifyStartup() {
+        await().atMost(5, MINUTES)
+                .pollInterval(5, SECONDS)
+                .until(() -> {
+                    final String bundleCount = ssh(MINION_CONTAINER.getSshAddress(), "bundle:list | grep SCV | wc -l");
+                    final String scvBundles = ssh(MINION_CONTAINER.getSshAddress(), "bundle:list | grep SCV");
+                    return bundleCount.contains("3") && scvBundles.contains("Dominion gRPC Impl") && !scvBundles.contains("JCEKS Impl");
+                });
+    }
+
+    private String ssh(final InetSocketAddress sshAddress, final String command) {
+        try (final SshClient sshClient = new SshClient(sshAddress, USERNAME, PASSWORD)) {
```

Comment:
Use `MINION_CONTAINER.ssh()` instead

Reply:
There exist a KarafShell class, which is used in the smoke-tests, but this will also trigger log:display for each command execution. I just wanted to check the command's output, nothing else.

Reply:
Not what I'm talking about. `MINION_CONTAINER.ssh()` is just `new SshClient(getSshAddress(), OpenNMSContainer.ADMIN_USER, OpenNMSContainer.ADMIN_PASSWORD)`...

Reply:
Not what I'm talking about. `MINION_CONTAINER.ssh()` is just `new SshClient(getSshAddress(), OpenNMSContainer.ADMIN_USER, OpenNMSContainer.ADMIN_PASSWORD)`...

Reply:
got it. done.